integration_tests/integration/book-recall.spec.js

```diff
@@ -104,15 +118,18 @@ context('Book a recall', () => {
     recallMissingDocuments.enterTextInInput({ name: 'missingDocumentsDetail', text: 'I sent an email' })
     recallMissingDocuments.uploadFile({ fieldName: 'missingDocumentsEmailFileName', fileName: 'email.msg' })
     recallMissingDocuments.clickContinue()
+    const checkAnswers = checkAnswersPage.verifyOnPage()
+    checkAnswers.clickContinue()
+    recallConfirmationPage.verifyOnPage({ personName })
+  })
+
+  it('User can check their answers', () => {
     // eslint-disable-next-line no-unused-vars
     const [licence, ...documents] = [...getRecallResponse.documents]
```

Comment:
Just wondering in passing why my IDE thinks `license` here is unused; presumably the // eslint... is also related; TBH I've not tried w/out; will do sometime.